### PR TITLE
Swaps the current ice cream machine for the NTStation Variant

### PR DESCRIPTION
**Pull request by Iamgoofball**
anzaf1t

Since we've overhauled chemistry and gotten up to exactly 420 reagents(HG found this out doing the April Fools joke this year) I felt this would be more useful

### CODE REVIEW

**Comment by sawu-tg** [comment]
:+1:

**Comment by Iamgoofball** [comment]
whoops forgot about metastation

@Metacide do you wanna make the map changes or can I

**Comment by Jordie0608** [comment]
What's different about the NT variant?

**Comment by Dorsisdwarf** [comment]
It makes any reagant into icecream

**Comment by vista-** [comment]
Emagging it should make you able to give an arbitrary name to the ice cream.

**Comment by WJohn** [comment]
I'm hoping we won't get super hideous neon blue and neon red ice cream colors.

**Comment by sawu-tg** [comment]
@WJohn If it still uses my original reagent coloring and naming, it will draw from the highest concentration reagent (ie >50% of the contained reagents) and use it's reagent color.

File: code/modules/food&drinks/food/snacks_other.dm [hunk 1]
```diff
@@ -187,4 +187,36 @@
...
+	filling.icon += mix_color_from_reagents(reagents.reagent_list)
+	overlays += filling
+	filling_color = mix_color_from_reagents(reagents.reagent_list)
+
+/obj/item/weapon/reagent_containers/food/snacks/icecream/icecreamcone
+	name = "ice cream cone"
+	desc = "Delicious ice cream."
+	icon_state = "icecream_cone"
+	volume = 500
+	list_reagents = list("nutriment" = 6, "sugar" = 2)
```
Comment (phil235) [inline code/modules/food&drinks/food/snacks_other.dm]: these two needs a filling_color

**Comment by Metacide** [comment]
If this is ready to go, merge it and I'll cancel https://github.com/tgstation/-tg-station/pull/8813 and make a new pull for that update.

**Comment by Metacide** [comment]
@Iamgoofball - If my update goes through, you're welcome to update accordingly afterwards, but it's only a tiny fix for a broken APC, so no worries if you need to plough through before conflicts get crazy instead.

**Comment by Iamgoofball** [comment]
@WJohn Plus it won't be neon blue/red anyways due to the base overlay's color setup.

**Comment by Iamgoofball** [comment]
>map merge conflicts

kill me now

**Comment by Fox-McCloud** [comment]
Initropidril flavored ice cream....delicious.

**Comment by Metacide** [comment]
Closed the MetaStation update, will add this to MetaStation whenever I next have time.

**Comment by Iamgoofball** [comment]
I can just add it in this PR.

**Comment by vista-** [comment]
Status on this?

**Comment by Iamgoofball** [comment]
dead due to map merge issues, thanks map merge

**Pull request closed without merging**